vfront_verilog_parser/src/include/tests.rs

```diff
@@ -0,0 +1,34 @@
+use super::*;
+use IncludeSearchMode::*;
+
+#[test]
+fn test_filesystem_include_provider() {
+    // TODO: this is testing using some existing workspace files rather than designated
```

Comment:
yes, don't do this — instead, add the `tempfile` crate to dev-dependencies, use it to create and fill out a temp dir (or several)